tests/queries/0_stateless/02161_addressToLineWithInlines.sql

```diff
@@ -1,4 +1,5 @@
--- Tags: no-tsan, no-asan, no-ubsan, no-msan, no-debug, no-cpu-aarch64
+-- Tags: no-tsan, no-asan, no-ubsan, no-msan, no-debug, no-cpu-aarch64, disabled
+-- Tag disabled: Parsing inlines may lead to "could not find abbreviation code" (FIXME)
```

Comment:
Do you plan to continue working on it?

Reply:
Yes, I have this in my todo list (and not the long term one)